js/index.js

```diff
@@ -51,3 +51,23 @@ closePopupBtn.addEventListener('click', () => {
   popupContainer.classList.remove('display-active');
   // body.style.overflowY = 'visible';
 });
+
+const email = document.getElementById('email');
+const form = document.getElementById('form');
```

Comment:
- [ ] The problem not resolved because you didn't get the the form
you can solve this by replace it with this line `const form = document.querySelector('.contact-form');`